azure/notebooks/Train-SKLearn.ipynb

```diff
@@ -206,7 +206,7 @@
     "\n",
     "These run metrics will become particularly important when we begin hyperparameter tuning our model in the \"Tune model hyperparameters\" section.\n",
     "\n",
-    "Once your script is ready, copy the training script `train_rapids_RF.py` into your project directory."
+    "Once your script is ready, copy the training script `train_sklearn_RF.py` into your project directory."
```

Comment:
Should this be train_sklearn.py?

Reply:
The script only includes CPU RF, planning to merge with rapids_csp_azure and add XGB in the next PR